actions/views/rhs.js

```diff
@@ -96,6 +96,18 @@ export function showSearchResults() {
     };
 }
 
+export function showRHSPlug(pluginId) {
```

Comment:
Could you rename this `showRHSPlugin`? I don't think we abbreviate plugin to plug anywhere else, so I'd prefer to keep it consistent, especially since this gets used by the plugins themselves

Reply:
Sure. I went back and forth on the now on this. I noticed that in the webapp they called "plug" the components where the plugins could "plug" their custom components: https://github.com/mattermost/mattermost-webapp/tree/master/plugins/channel_header_plug

But it makes sense that this is exposed to be used by the plugins themselves.

Reply:
Maybe @crspeller or someone else on the extensions team might have more of an opinion on this naming, but my personal preference is just to avoid abbreviations.

Reply:
I think `showRHSPlugin` makes more sense to me. (1/5)